fileSummaryV1/metadata_api.js

```diff
@@ -61,25 +63,26 @@ function getGsUriFromDataObject(dataObjectMetadata) {
 }
 
 function getDataObjectMetadata(dataObjectUri) {
-    const newUri = helpers.dataObjectUriToHttps(dataObjectUri);
+    const newUri = dataObjectUriToHttps(dataObjectUri);
 
     return apiAdapter.getJsonFrom(newUri)
         .then((response) => response.data_object)
         .then((metadata) => {
-            const { mime_type, size, created, updated, checksums } = metadata;
+            const { mimeType, size, created, updated, checksums } = metadata;
             const gsUri = getGsUriFromDataObject(metadata);
             const [bucket, name] = parseGsUri(gsUri);
 
-            return {
-                contentType: mime_type || 'application/octet-stream',
+            return convertToFileInfoResponse(
+                mimeType || 'application/octet-stream',
                 size,
-                timeCreated: created ? new Date(created).toString() : undefined,
-                updated: updated ? new Date(updated).toString() : undefined,
-                md5Hash: (checksums.find((e) => e.type === 'md5') || {}).checksum,
+                created ? new Date(created).toString() : null,
+                updated ? new Date(updated).toString() : null,
+                (checksums.find((e) => e.type === 'md5') || {}).checksum,
                 bucket,
                 name,
-                gsUri
-            };
+                gsUri,
+              null
```

Comment:
Newbie JS question: What's the difference between passing `null` here and `undefined` in the tests?

Also, does the linter prefer this different indentation?

Reply:
I don't honestly know the difference between `null` and `undefined`.  Codacy was complaining about the use of `undefined`, so I changed it. 🤷

Reply:
The different indentation was a mistake — fixed.

Reply:
Does codacy review `test` code? Would it hurt to also pass `null` in the tests too?

Reply:
Nevermind, you're already ahead of me